doc/guides/collaborator-guide.md

```diff
@@ -143,30 +143,28 @@ the TSC meeting agenda.
 
 ### Waiting for Approvals
 
-Before landing pull requests, allow 48 hours for input from other Collaborators.
-Certain types of pull requests can be fast-tracked and may land after a shorter
-delay. For example:
+Before landing pull requests, allow for input from other Collaborators.
+If you expect that a pull request might raise objections, it is a good idea to
+wait longer before landing. You can also add the `wait-for-feedback` label
+to pull requests; in that case, it is expected that the pull request will not
+be merged until at least 48 hours after it has been opened, in order to allow
+for everybody to comment on it.
 
-* Focused changes that affect only documentation and/or the test suite:
-  * `code-and-learn` tasks often fall into this category.
-  * `good-first-issue` pull requests may also be suitable.
-* Changes that fix regressions:
-  * Regressions that break the workflow (red CI or broken compilation).
-  * Regressions that happen right before a release, or reported soon after.
+For pull requests that are lacking an approval from a member of the relevant
+team (e.g. @nodejs/http in the case of HTTP changes), it is recommended to keep
+them open for a longer period of time, e.g. several days.
 
-To propose fast-tracking a pull request, apply the `fast-track` label. Then add
-a comment that Collaborators may upvote.
+### Reverting PRs shortly after landing
 
-If someone disagrees with the fast-tracking request, remove the label. Do not
-fast-track the pull request in that case.
+If it has been less than 48 hours after a pull request has been opened and it
+has already been landed, and you object to it in a way that cannot be addressed
+through a new pull request which addresses your concerns, you can open a
+pull request containing a revert and merge it without waiting for approvals or
+CI runs.
```

Comment:
@nodejs/releasers we'll need to keep an eye on this and adapt release tooling if reverts become more present

Reply:
Agreed. Based on what I've seen in the Release WG mentoring sessions where we've been walked through the triage process of whether something is ready to land on LTS release branches it is already difficult to ascertain links between a PR and any subsequent reverts/fixups.